prow/plugins/config.go

```diff
@@ -466,6 +467,21 @@ type Welcome struct {
 	MessageTemplate string `json:"message_template,omitempty"`
 }
 
+// Dco is config for the DCO (https://developercertificate.org/) checker plugin.
+type Dco struct {
+	// Repos is either of the form org/repos or just org.
+	Repos []string `json:"repos,omitempty"`
+	// SkipDCOCheck is used to skip DCO check for collaborators and org members
+	SkipDCOCheck bool `json:"skip_dco_check,omitempty"`
+	// TrustedOrg is the org whose members' commits will not be checked for DCO signoff
+	// if the skip DCO option is enabled. The default is the PR's org.
+	TrustedOrg string `json:"trusted_org,omitempty"`
+	// OnlyOrgMembers requires PRs from non-org members to have a valid DCO signoff message
+	// if the skip DCO option is enabled.
+	// By default, SkipDCOCheck also includes repo collaborators.
+	OnlyOrgMembers bool `json:"only_org_members,omitempty"`
```

Comment:
Instead of having one bool that skips for both collaborator/member and then another bool that if the first bool is enabled re-enables not skipping for collaborators - Can we just have one bool `SkipDCOCheckForCollaborators` and another `SkipDCOCheckForMembers`?